include/mbedtls/config.h

```diff
@@ -2567,6 +2578,28 @@
  */
 #define MBEDTLS_XTEA_C
 
+/**
+ * \def MBEDTLS_SERIALIZE_C
+ *
+ * Enable the serialization module. This is needed for offloading (see
+ * MBEDTLS_FS_IO_ALT).
+ *
+ * Module:  library/serialize.c
+ * Caller:  library/net_sockets/serialize.c
+ */
+//#define MBEDTLS_SERIALIZE_C
+
+/**
+ * \def MBEDTLS_SERIALIZE_FORK_FRONTEND_C
+ *
+ * Enable testing of serialization module on a host machine using a frontend.
+ *
+ * Module:  library/serialize.c
+ * Caller:  library/net_sockets/serialize.c
+ *          library/fsio.c
+ */
```

Comment:
This requires `MBEDTLS_SERIALIZE_C`. Please add a check to `check_config.h`, too.